docs/reference/setup/advanced-configuration.asciidoc

```diff
@@ -110,35 +110,16 @@ NOTE: When running in a container, such as <<docker,Docker>>, total memory is
 defined as the amount of memory visible to the container, not the total system
 memory on the host.
 
-* Set `Xms` and `Xmx` to no more than 32 GB, the approximate threshold for
-compressed ordinary object pointers (oops). To verify you are under the
+* Set `Xms` and `Xmx` to no more than the threshold for compressed ordinary
+object pointers (oops). The exact threshold varies but 26GB is safe on most
+systems and can be as large as 30GB on some systems. To verify you are under the
 threshold, check `elasticsearch.logs` for an entry like this:
```

Comment:
while we're here, `elasticsearch.logs` isn't anything right?
```suggestion
threshold, check the {es} log for an entry like this:
```

Reply:
Thanks for catching this. This should have been `elasticsearch.log` originally, which is in `$ES_HOME/logs` for our tar archive distros. I like {es} logs better though.